fetch.bs

```diff
@@ -5343,6 +5377,13 @@ method, must run these steps:
  {{Headers}} object whose <a for=Headers>guard</a> is
  "<code>immutable</code>".
 
+ <li><p>Add the following steps to <var>requestObject</var>'s <a for=Request>signal</a>'s
+ <a>cancelation callbacks</a>:
+
+ <ol>
+  <li><a lt=terminated for=fetch>Terminate the request</a>
```

Comment:
Yeah, I was hoping we could fix that a bit as part of doing this, but I don't have great suggestions as to how.